src/cmd/run/run.go

```diff
@@ -70,26 +74,53 @@ func run(ctx context.Context, names []string, options Options) error {
 		logging.Log("using image: %s", yeyContext.Image)
 	}
 
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "context:\n--\n%v--\n", yeyContext)
+	}
+
+	// Container name
 	containerName := yey.ContainerName(contexts.Path, yeyContext)
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "container: %s\n", containerName)
+	}
 
+	// Reset
 	if options.Reset {
+		if options.Verbose {
+			fmt.Fprintf(os.Stderr, "removing container first")
+		}
 		if err := docker.Remove(ctx, containerName); err != nil {
 			return fmt.Errorf("failed to remove container %q: %w", containerName, err)
 		}
 	}
 
+	var runOptions []docker.RunOption
+
+	// Working directory
 	workDir, err := getContainerWorkDir(yeyContext)
 	if err != nil {
 		return err
 	}
-
-	var runOptions []docker.RunOption
 	if workDir != "" {
 		runOptions = append(runOptions, docker.WithWorkDir(workDir))
 	}
+	if options.Verbose {
+		fmt.Fprintf(os.Stderr, "working directory: %s", workDir)
+	}
 
-	if err := ShowBanner(yeyContext.Name); err != nil {
-		return err
+	// Dry-run/verbose
+	runOptions = append(runOptions,
+		docker.WithDryRun(options.DryRun),
+		docker.WithVerbose(options.Verbose))
+
+	// Banner
+	if options.Verbose {
+		fmt.Fprintln(os.Stderr)
+	}
+	if !options.DryRun {
```

Comment:
Dry-run does not print the banner but still starts the docker process? Wouldn't the flag be `--no-banner`?

Reply:
nvm